package.json

```diff
@@ -1,72 +1,60 @@
 {
-  "name": "@ricado/api-client",
-  "version": "0.1.49",
-  "description": "RICADO Gen 4 API Client Library for NodeJS and Browsers",
-  "author": {
-    "name": "Ash Neilson"
-  },
-  "main": "./dist/es5/index.js",
-  "types": "./dist/index.d.ts",
-  "files": [
-    "dist",
-    "src",
-    "types"
-  ],
-  "browser": {},
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/ricado-group/js-api-client.git"
-  },
-  "bugs": {
-    "url": "https://github.com/ricado-group/js-api-client/issues"
-  },
-  "license": "MIT",
-  "keywords": [
-    "ricado",
-    "gen4",
-    "api",
-    "client"
-  ],
-  "scripts": {
-    "clean": "rimraf dist",
-    "test": "cross-env BABEL_ENV=commonjs mocha --require babel-core/register --recursive",
-    "test:watch": "npm test -- --watch",
-    "cover": "cross-env BABEL_ENV=commonjs istanbul cover _mocha -- --require babel-core/register --recursive",
-    "lint": "eslint --fix src test",
-    "build": "genversion --semi --es6 src/PackageVersion.js && cross-env BABEL_ENV=commonjs babel src --out-dir dist/es5",
-    "build:watch": "./node_modules/.bin/watch \"npm run build\" src",
-    "build:bundle": "npm run build && cross-env NODE_ENV=production webpack -p --config webpack.config.prod.js",
-    "docs": "./node_modules/.bin/jsdoc -c jsdoc.conf.json",
-    "prepublishOnly": "npm run clean && npm run build:bundle && npm run docs",
-    "types-lint": "tsc && dtslint --localTs node_modules/typescript/lib types"
-  },
-  "dependencies": {
-    "node-fetch": "^2.6.1",
-    "socket.io-client": "^2.3.0"
-  },
-  "devDependencies": {
-    "@microsoft/rush-stack-compiler-3.5": "0.4.3",
-    "babel-cli": "^6.26.0",
-    "babel-eslint": "^8.2.6",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-preset-env": "^1.7.0",
-    "chai": "^4.2.0",
-    "cross-env": "^5.2.1",
-    "dts-bundle": "^0.7.3",
-    "dtslint": "^3.6.12",
-    "eslint": "^4.19.1",
-    "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-plugin-import": "^2.22.0",
-    "genversion": "^2.2.1",
-    "ink-docstrap": "^1.3.2",
-    "istanbul": "^0.4.5",
-    "jsdoc": "^3.6.4",
-    "jsdoc-memberof-namespace": "^2.2.0",
-    "mocha": "^5.2.0",
-    "rimraf": "^2.7.1",
-    "typescript": "^3.9.6",
-    "watch": "^1.0.2",
-    "webpack": "^4.20.2",
-    "webpack-cli": "^3.1.1"
-  }
+    "name": "@ricado/api-client",
+    "version": "0.1.49",
+    "description": "RICADO Gen 4 API Client Library for NodeJS and Browsers",
+    "author": {
+        "name": "Ash Neilson"
+    },
+    "main": "./dist/index.js",
+    "module": "./src/index.js",
```

Comment:
I'm getting an error `Support for the experimental syntax 'classProperties' isn't currently enabled` due to the fact that the CRA webpack is selecting the "module" field as opposed to the compiled "main". I assume that node_modules aren't included in the babel tranformations for node_module packages. I can look into this more if you like?

Reply:
@sadsa Ah yes this is still in ESNext and not into any completed standard yet :(

I think we either remove the "module" field for now which forces everyone to use the "dist" pre-built code. This is transpiled by babel to ES2019 (I think).

We must define the Static Class Properties otherwise Type-Script Checks fail as the property is never defined. I could re-write as a Class that can be initialized and make it a singleton - but I prefer the static class.

I can remove the module definition for now or point to /dist/index.js if you agree? 👍

Reply:
@sadsa Updated `package.json` and removed the module definition.